packages/notifi-react-card/lib/components/subscription/subscription-card-views/preview-panel/UserInfoPanel.tsx

```diff
@@ -50,6 +76,26 @@ export const UserInfoPanel: React.FC<UserInfoPanelProps> = ({
           >
             {email ?? 'email'}
           </label>
+          {!isEmailConfirmed ? (
```

Comment:
nit: i'd invert the conditional (so the state exposes a `emailNeedsConfirmation`)